build-farm/platform-specific-configurations/aix.sh

```diff
@@ -18,6 +18,14 @@
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+# Clear out /tmp/sh-np files if we're running in jenkins to avoid
+# "ambiguous redirect" pipe failures. This will be ok with only one executor
+# See https://github.com/adoptium/infrastructure/issues/929
+if [ "$(whoami)" = "jenkins" ] && [ -n "$WORKSPACE" ]; then
```

Comment:
```suggestion
if [ "$(whoami)" == "jenkins" ] && [ -n "$WORKSPACE" ]; then
```

Reply:
> i think the == is required

No, single equals is standard POSIX shell syntax for string equality.